src/instructlab/utils.py

```diff
@@ -433,11 +441,13 @@ def validate_taxonomy_file(file_path: str, yaml_rules: Optional[str] = None):
             errors += 1
             return warnings, errors
 
-        # do general YAML linting if specified
+        # do general YAML linting
         version = get_version(contents)
         if version > 1:  # no linting for version 1 yaml
-            if yaml_rules is not None:
-                if os.path.isfile(yaml_rules):
+            if yaml_rules is not None:  # user attempted to pass custom yaml rules file
+                if os.path.isfile(
```

Comment:
should we fail is yaml rules is not none and this isfile check is false?

Reply:
Hmmm I see your point - we have the debug statement below, but maybe we could make it into a warning instead?

Reply:
Come to think of it, the `debug` statement when the file _is_ found could maybe be `info`?

Reply:
This code will all be removed and replaced by calling the new taxonomy parse API in instructlab-schema. So I would not worry too much about this here.